vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -55,6 +55,8 @@
     public String clientSecret;
     public String scope;
 
+    public String accessToken;
```

Comment:
When I was implementing the access token I had the same thought (and moreover, that the builder could probably be slimmed down a bit) so I added a ticket to the tech debt sheet around 1) fixing up the builder and 2) making our vimeoaccount/auth header generation a little more transparent and mutable.